recipes/default.rb

```diff
@@ -15,9 +15,24 @@
 
 package 'nzbdrone'
 
-template '/etc/init/sonarr.conf' do
-  source 'init.conf.erb'
-  variables user: node[:sonarr][:user]
+case node['platform']
+when 'ubuntu'
+  if node['platform_version'].to_f >= 15.04
+	template '/etc/systemd/system/sonarr.service' do
+	  source 'sonarr.service.erb'
+	  variables user: node[:sonarr][:user]
+	end
```

Comment:
Could you change indenting to 4 spacers here (indenting two from the `if` above)?